src/com/xilinx/rapidwright/design/DesignTools.java

```diff
@@ -2048,7 +2048,16 @@ private static void copySiteRouting(Cell copy, Cell orig, Map<String,String> src
 									curr = tmp;
 									break;
 								}
-							}	
+							}
+							if(rtCopy.getBELName().endsWith("6LUT")) {
```

Comment:
Maybe looking too far ahead, but would this be necessary for Versal? AM005 states:
> Enables dual LUT functions of up to six inputs (five in prior architectures)

Reply:
Yes, this probably only applies to UltraScale{,+}, I've added a check.